### PR TITLE
Update README.md with new RSS functionalities and clean up

### PR DESCRIPTION
**Pull request by JaGeo**
Hi @vlderinger @YuanbinLiu @QuantumChemist , 

I started with the pull request. I am happy to work together on improving the README.md to cover the RSS functionality in a better way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,39 +59,6 @@ A short guide to contributing to autoplex can be found [here](https://autoatml.g
...
-The workflow starts with three flows that generate data for our database:
-* The first flow is preparing the VASP calculation for the isolated atoms (`isolated atoms calcs`).
-* A second flow is preparing the `phonopy` calculations to collect the VASP data from the single-atom displaced supercells (`phonopy based structures`).
-* The third flow is constructing rattled supercells by rattling the atoms, i.e. displacing all atoms' positions (in the default setup), preparing the VASP calculations and collecting the data for the MLIP fit (`rattled DFT structures`).
-
-After a few data preprocessing steps (`data preprocessing`) to filter out data with too strong force values, the MLIP fit (`machine learning fit`) is run and the resulting potential is used for the benchmark against DFT data (`benchmark`).
-Finally, the result metrics are collected in form of output plots and files (`write benchmark metrics`).
+We currently have two different types of workflows available. 
+* Workflow to use random-structure searches for the creation for systematic construction of interatomic potentials
+* Workflow to train accurate interatomic potentials for harmonic phonon properties
```
Comment (JaGeo) [inline README.md:64]: @vlderinger Fine like this?

File: README.md [hunk 2]
```diff
@@ -59,39 +59,6 @@ A short guide to contributing to autoplex can be found [here](https://autoatml.g
...
-The workflow starts with three flows that generate data for our database:
-* The first flow is preparing the VASP calculation for the isolated atoms (`isolated atoms calcs`).
-* A second flow is preparing the `phonopy` calculations to collect the VASP data from the single-atom displaced supercells (`phonopy based structures`).
-* The third flow is constructing rattled supercells by rattling the atoms, i.e. displacing all atoms' positions (in the default setup), preparing the VASP calculations and collecting the data for the MLIP fit (`rattled DFT structures`).
-
-After a few data preprocessing steps (`data preprocessing`) to filter out data with too strong force values, the MLIP fit (`machine learning fit`) is run and the resulting potential is used for the benchmark against DFT data (`benchmark`).
-Finally, the result metrics are collected in form of output plots and files (`write benchmark metrics`).
+We currently have two different types of workflows available. 
+* Workflow to use random-structure searches for the systematic construction of interatomic potentials
+* Workflow to train accurate interatomic potentials for harmonic phonon properties
```
Comment (JaGeo) [inline README.md:64]: @vlderinger Sorry again. Fine?

Comment (vlderinger) [reply README.md:64]: Sounds great! We could say "iterative random structure searching (RSS)" to make it clearer that the process runs over many loops – this is where the workflows are particularly useful. @YuanbinLiu are you happy with the above wording too?

I would also link here to the key publications for the ideas that are implemented in the respective workflows – https://doi.org/10.1103/PhysRevLett.120.156001 and https://doi.org/10.1038/s41524-019-0236-6 for GAP-RSS, and https://doi.org/10.1063/5.0013826 for phonons.

**Comment by QuantumChemist** [comment]
Would you mind if I smuggled in here the one commit I want to make to add to the documentation that the jobnames within autoplex will be fixed?

**Comment by JaGeo** [comment]
@QuantumChemist sure

**Comment by QuantumChemist** [comment]
> @QuantumChemist i would not. Go ahead

Okidki 😃

**Comment by JaGeo** [comment]
@QuantumChemist, Could you (next week) please add a disclaimer to the README.md stating that we currently expect familiarity with jobflow, atomate2, and jobflow-remote/fireworks and that people should, for now, refer to the respective documentation for further information?

File: README.md (hunk 2 above)
Comment (YuanbinLiu) [reply README.md:64]: Yes, sounds great to me!

**Comment by QuantumChemist** [comment]
> @QuantumChemist, Could you (next week) please add a disclaimer to the README.md stating that we currently expect familiarity with jobflow, atomate2, and jobflow-remote/fireworks and that people should, for now, refer to the respective documentation for further information? We could also add this to the documentation

I will add all the commits then to this PR directly!

File: README.md (hunk 2 above)
Comment (JaGeo) [reply README.md]: @QuantumChemist Could you add the references mentioned above to the readme as well? :). Thanks in advance.

File: README.md [hunk 3]
```diff
@@ -58,40 +71,7 @@ curl -O https://www.mtg.msm.cam.ac.uk/files/airss-0.9.3.tgz; tar -xf airss-0.9.3
...
-
-After a few data preprocessing steps (`data preprocessing`) to filter out data with too strong force values, the MLIP fit (`machine learning fit`) is run and the resulting potential is used for the benchmark against DFT data (`benchmark`).
-Finally, the result metrics are collected in form of output plots and files (`write benchmark metrics`).
+<!-- linter-disable spelling -->
+We currently have two different types of automation workflows available:
+* Workflow to use random-structure searches for the systematic construction of interatomic potentials. The idea is based on the articles [*Phys. Rev. Lett.* **120**, 156001 (2018)](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.156001) and [*npj Comput. Mater.* **5**, 99 (2019)](https://www.nature.com/articles/s41524-019-0236-6).
+* Workflow to train accurate interatomic potentials for harmonic phonon properties. The idea is based on [*J. Chem. Phys.* **153**, 044104 (2020)](https://pubs.aip.org/aip/jcp/article/153/4/044104/1056348/Combining-phonon-accuracy-with-high).
```
Comment (JaGeo) [inline README.md:77]:
```suggestion
We currently have two different types of automation workflows available:
* Workflow to use random-structure searches for the systematic construction of interatomic potentials. The implementation automates ideas from the following articles: [*Phys. Rev. Lett.* **120**, 156001 (2018)](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.156001) and [*npj Comput. Mater.* **5**, 99 (2019)](https://www.nature.com/articles/s41524-019-0236-6).
* Workflow to train accurate interatomic potentials for harmonic phonon properties. The implementation automates the ideas from the following article: [*J. Chem. Phys.* **153**, 044104 (2020)](https://pubs.aip.org/aip/jcp/article/153/4/044104/1056348/Combining-phonon-accuracy-with-high).
```

Comment (JaGeo) [reply README.md:77]: @QuantumChemist maybe like this?

**Comment by QuantumChemist** [comment]
Nothing worked, except for shortening it further to "Mat." 🙈 
@JaGeo , if you don't have any more things to improve and change, I would say that the PR is ready to be merged? 👀

**Pull request merged**